package.json

```diff
@@ -10,8 +10,19 @@
     "setup": "npm install",
     "postsetup": "npm run build"
   },
+  "dependencies": {
```

Comment:
Why are these dependencies here? To my unnderstanding, there is no code in the outer package that needs these dependencies to run

Reply:
Ok